### PR TITLE
PS-4573 5.7 Patch breaks creates .cnf file parsing problem

### PR DESCRIPTION
**Pull request by patrickbirch**
Added text to the yum_repo.

### CODE REVIEW

**Comment by patrickbirch** [comment]
waiting for comments or approval.

File: doc/source/installation/yum_repo.rst
Comment (tplavcic) [inline doc/source/installation/yum_repo.rst]: a bit strange `installs and updates creates a mysqld.cnf` shouldn't it be `installs and updates a mysqld.cnf` or something similar?

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: doc/source/installation/yum_repo.rst
Comment (patrickbirch) [reply doc/source/installation/yum_repo.rst]: Rewrote the paragraph.

**Review by tplavcic — APPROVED** [review]
LGTM

**Pull request merged**
